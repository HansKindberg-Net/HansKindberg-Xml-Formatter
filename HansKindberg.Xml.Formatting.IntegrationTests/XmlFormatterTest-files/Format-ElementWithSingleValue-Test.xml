<?xml version="1.0" encoding="utf-8"?>
<root><elementWithSingleValue>Text, text, text, text </elementWithSingleValue><element>Text, text, text, text <!--Comment-->Text, text, text, text <!--Comment--></element></root>