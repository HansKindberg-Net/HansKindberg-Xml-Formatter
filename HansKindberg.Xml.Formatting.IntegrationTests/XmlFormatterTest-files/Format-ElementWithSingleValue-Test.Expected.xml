<?xml version="1.0" encoding="utf-8"?>
<root>
	<element>
		Text, text, text, text 
		<!-- Comment -->
		Text, text, text, text 
		<!-- Comment -->
	</element>
	<elementWithSingleValue>Text, text, text, text </elementWithSingleValue>
</root>